src/shared/prerendered-app/Intro/index.tsx

```diff
@@ -240,15 +246,10 @@ export default class Intro extends Component<Props, State> {
               class={style.blobCanvas}
             />
           )}
-          <h1 class={style.logoContainer}>
-            <img
-              class={style.logo}
-              src={logoWithText}
-              alt="Squoosh"
-              width="539"
-              height="162"
-            />
-          </h1>
+          <h1
+            class={style.logoContainer}
+            dangerouslySetInnerHTML={{ __html: logoSVGSource }}
+          />
```

Comment:
(copying this from chat):

Another option, and one that avoids innerHTML, would be to use `<use>` here:
https://codepen.io/developit/pen/WNOKwGZ?editors=1000

The generated HTML would contain a hidden SVG with the logo defined as a symbol:

```html
<svg style="display:none;">
	<defs>
		<symbol id="logo" width="270" height="81">
			<!-- contents of SVG file here -->
		</symbol>
	</defs>
</svg>
```

... and then this line becomes:

```suggestion
          <h1 class={style.logoContainer}>
              <svg viewBox="0 0 269.5 81">
                  <use href="#logo" />
              </svg>
          </h1>
```

Reply:
The downside of this is I'd need to move the `<defs>` up into the `static-build` (since those files are only used to generate the HTML). It creates an interdependency, but I don't know if that's worse than the 'trick' going on here. WDYT @surma?